### PR TITLE
Add note about ShaderViewportIndexLayerEXT being split up

### PR DESCRIPTION
**Pull request by sjfricke**
closes https://github.com/KhronosGroup/Vulkan-Docs/issues/1948

### CODE REVIEW

File: appendices/versions.adoc [hunk 1]
```diff
@@ -206,6 +206,9 @@ via flink:vkGetPhysicalDeviceFeatures2.
 
 === Differences relative to `VK_EXT_shader_viewport_index_layer`
 
+The code:ShaderViewportIndexLayerEXT SPIR-V capability was replaced the with
+the code:ShaderViewportIndex and code:ShaderLayer capability. Declaring both
```
Comment (r-potter) [inline appendices/versions.adoc:210]: 'the with the'

Otherwise functionally correct

File: appendices/versions.adoc [hunk 2]
```diff
@@ -206,6 +206,9 @@ via flink:vkGetPhysicalDeviceFeatures2.
 
 === Differences relative to `VK_EXT_shader_viewport_index_layer`
 
+The code:ShaderViewportIndexLayerEXT SPIR-V capability was replaced the with
```
Comment (dgkoch) [inline appendices/versions.adoc:209]:
```suggestion
The code:ShaderViewportIndexLayerEXT SPIR-V capability was replaced with
```

File: appendices/versions.adoc (hunk 1 above)
Comment (oddhack) [reply appendices/versions.adoc:210]: Also 'capabilities' at the end of sentence, not 'capability'.

**Comment by oddhack** [comment]
Signed off w/minor grammar fixes.

File: appendices/versions.adoc (hunk 1 above)
Comment (sjfricke) [reply appendices/versions.adoc:210]:
```suggestion
the code:ShaderViewportIndex and code:ShaderLayer capabilities. Declaring both
```

**Pull request merged**